### ISSUE TITLE
error: vortex crashing on start

### ISSUE THREAD

**Issue by WalkerSpectre** [Nexus-Mods/Vortex#7898]
Basically I've worked with vortex for a long time and suddenly, even though nothing changed in my system, vortex crashes when it starts. It shows the vortex logo, and as soon as it starts loading it crashes. Vortex only appears in the task manager for about 3 seconds before crashing.

I really don't know how to reproduce the error, it happened spontaneously and I don't know what caused it, since it didn't update and nothing else changed on my system.

I can't screenshot something NOT happening, so yeah...

**Platform**
 - OS: Windows 10
 - Vortex Version 1.3.17

**Comment by IDCs** [comment]
Hi @WalkerSpectre, for us to help you we're going to need a few files - can you have a look inside your "%appdata%/Vortex" directory and attach all of your vortex.log files.

Additionally have a look inside "%appdata%/Vortex/temp/dumps" and attach any files you find in there (if any)

**Comment by WalkerSpectre** [comment]
> Hi @WalkerSpectre, for us to help you we're going to need a few files - can you have a look inside your "%appdata%/Vortex" directory and attach all of your vortex.log files to this issue.
> 
> Additionally have a look inside "%appdata%/Vortex/temp/dumps" and attach any files you find in there (if any)



 
logs here, nothing on dumps

**Comment by IDCs** [comment]
Thanks @WalkerSpectre, unfortunately those logs do not seem to have captured the crash/error. 
Does this crash happen every time you start up the application ?
Is there any error being reported at any time or does the application basically just never starts up ?

Lets try something - please go to your "C:\Program Files\Black Tree Gaming Ltd\Vortex\resources\app.asar.unpacked\bundledPlugins" folder and temporarily move the "gamebryo-plugin-management" folder somewhere else (desktop is fine) - does Vortex start up successfully without the plugin management extension ?

**Comment by WalkerSpectre** [comment]
It does not start up correctly once i do that, it says the files have been corrupted. It does happen every time, but I used vortex like 2 days ago and it worked fine, and no it doesn't show any error message once it crashes.

**Comment by IDCs** [comment]
Hi @WalkerSpectre, yeah I forgot to mention it will report corruption (it notices that the expected Vortex setup is incorrect) but it should allow you to continue - click continue and see if the application manages to start up.

**Comment by WalkerSpectre** [comment]
Hello again, it did work without that folder. I didn't even notice the "ignore" option the first time, my bad. Still, is this a permanent fix?

**Comment by IDCs** [comment]
Thanks @WalkerSpectre, not a permanent fix no, just wanted to confirm a hunch - feel free to re-instate the folder while we look into reasons why this could be happening.

**Comment by TanninOne** [comment]
@WalkerSpectre What is the last version you can confirm worked correctly?

**Comment by WalkerSpectre** [comment]
@TanninOne supposedly the same version that's not working (1.3.17) because i dont remember having autoupdate activated and before I made this report it was working fine. It stopped working spontaneously.

**Comment by IDCs** [comment]
@WalkerSpectre - we suspect that your issue is caused by one or more of your plugins - can you please go to your "../fallout4/data/" folder and move any mod .esp files you find there (leave the native plugins alone) to a different temporary folder and try to start Vortex again and let us know if you're still getting the issue (if you haven't restored the gamebryo-plugin-management folder, please do that before you run Vortex)

If it starts working again - can you please restore each plugin one-by-one until Vortex crashes again so we can pinpoint the problematic plugin file ?

**Comment by WalkerSpectre** [comment]
@IDCs Excuse me for my ignorance but is it the proper game folder (inside steamapps) or the folder in vortex? Because in vortex there isnt any "data" and in the steamapps folder, inside fallout4, there is a data folder but no esp files.

**Comment by WalkerSpectre** [comment]
@IDCs btw it suddenly started working again but it stopped working 10 seconds later

**Comment by IDCs** [comment]
Hi @WalkerSpectre, sorry for replying so late - can you please update to 1.3.18 and see if you still get this issue ?

**Comment by WalkerSpectre** [comment]
@IDCs okay, thank you. I will update and i will report it if anything unusual happens.

**Comment by WalkerSpectre** [comment]
@IDCs Hey i just updated and it's still happening, i would do as you said and submit it through the internal feedback system, but how can i use it if Vortex won't open? XD I'll send it through here if you don't mind.

**Comment by WalkerSpectre** [comment]
@IDCs hey again, dont wanna be annoying but i still have the same problem. Again, I can't report it on the feedback system because the app won't open (that's the whole point...), so I'm still waiting.
Hope you answer this soon, I've been waiting for 9 days.

**Comment by IDCs** [comment]
Hi @WalkerSpectre, apologies for not responding sooner - I know you've been waiting for a solution to this issue for a while now, but unfortunately we're still unable to pinpoint why this is happening as the logs do not give us anything to go on.

I assume you've tried to re-install Vortex with no luck ?

**Comment by IDCs** [comment]
@WalkerSpectre Ok I just noticed a few log entries in your oldest log which may be worth investigating:

Sun, 08 Nov 2020 15:23:49 GMT - debug: [gamebryo-plugin-management] synching plugins pluginsPath=C:\Users\gcsar\AppData\Local\Fallout4
Sun, 08 Nov 2020 15:23:49 GMT - error: [gamebryo-plugin-management] failed to look for plugin changes pluginPath=C:\Users\gcsar\AppData\Local\Fallout4, errno=-4058, syscall=watch, code=ENOENT, path=C:\Users\gcsar\AppData\Local\Fallout4, filename=C:\Users\gcsar\AppData\Local\Fallout4

It appears that your "C:\Users\gcsar\AppData\Local\Fallout4" directory was missing at one point which blocked Vortex from deserializing your plugins list. Can you please confirm that directory exists and that the plugins.txt file is accessible ?

**Comment by WalkerSpectre** [comment]
sorry for the wait, i thought you gave up on the discussion so i didnt come check earlier.
No, there is no plugins.txt. Maybe that's the problem?

**Comment by IDCs** [comment]
Hi @WalkerSpectre, no worries - this issue is just annoying enough to always be in the back of my mind - so no, I didn't give up :)

Ok - so Vortex will automatically create both the file path and the plugins.txt file even if you remove them manually (or should do so) which leads me to believe that your problem is connected to the fact that Vortex is _trying_ to create the file but something in your environment is blocking it.

Usually Vortex will quickly get back to the user and either report an error or raise a UAC dialog if it doesn't have sufficient permissions to create the file; so what I think is happening in your case is - Vortex tried/tries to raise a UAC dialog and is waiting for you to approve or deny elevation but I'm guessing you never see the dialog pop up, leading to a perpetual loading screen. 

Are you perhaps aware of any permission related issues/modifications on your system ? Did you perhaps turn off UAC ?

**Comment by WalkerSpectre** [comment]
I created the two empty .txt files, and when i tried to launch vortex it led to something different. Instead of having just a flash of the loading screen, it actually loaded everything but it crashed afterwards.

A second after, i tried again, and it opened vortex correctly but after 3 seconds or so it crashed again, but i believe i saw a error message flashing (it opened a window and closed immediately, so i couldn't read it).

Again, i'm sorry this issue is so annoying for you, it's annoying for me too. I make mods for Fallout 4 (and i also enjoy the game lmao) and i haven't been able to play it because of the mods loaded.

Thank you for the help, though.

**Comment by WalkerSpectre** [comment]
I managed to record the event and i managed to pinpoint the specific frame, and it said:

"The LOOT userlist "C:\Users\gcsar\AppData\Roaming\Vortex\fallout4\userlist.yami" can't be read.

You should quit vortex now and repair the file.
If (and only if) you didn't modify the file yourself, please send in a bug report with that file attached.

If you continue now, the userlist will be reset and all your plugin rules and group assignments will be lost"

So basically it tells me to send the file in a bug report, but since i cant do it i would send it here, but github doesn't support the file. By the way, it presented two options: reset userlist and Quit Vortex, but since the latter was selected before dissapearing i imagine it's the one that was ultimately selected.

**Comment by TanninOne** [comment]
Yes, Vortex quits without resetting the userlist.

You can of course zip the file and attach it here.

**Comment by WalkerSpectre** [comment]
If i reset the userlist wont it erase all my plugins and stuff?
[userlist.zip](https://github.com/Nexus-Mods/Vortex/files/5717339/userlist.zip)

**Comment by TanninOne** [comment]
It will just reset your custom plugin rules, but that file is completely screwed anyway, there is no way to salvage that. So yeah, you'll have to delete that file.

**Comment by WalkerSpectre** [comment]
ok i think vortex is working almost perfectly now, so i have just one more question:

I opened fallout 4, and all the mods are supposedly working according to vortex, but the game save says there are mods missing (despite me checking if they were there).

how can i solve this?

**Comment by IDCs** [comment]
I'm assuming you have confirmed that the mods are indeed present and are deployed to the game's directory (Steam/Steamapps/common/etc..) - AFAIK the game will only complain about missing plugins if there's a discrepancy between the plugins that were enabled at the time when you created the save game and the plugins.txt file which you've created manually.

Given that Vortex now functions correctly, that file should've been populated using your plugins page - have a look through the plugins file and see whether the plugins/mods that the game says are missing are located inside that file.

If they are - chances are the game got confused by the fact that you created those manually. In which case - you should backup your saves just in case - and then resume the game (ignoring the missing mods warning); and see if the plugin truly is missing.

**Comment by IDCs** [comment]
Assumed resolved.
